packages.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Get list of packages
+  kubernetes.core.k8s_info:
+    api_version: "packages.operators.coreos.com/v1"
+    kind: PackageManifest
+    namespace: '{{ catalog_source_namespace | default("openshift-marketplace") }}'
+    label_selectors:
+      - 'catalog = {{ catalog_source | default("certified-operators") }}'
+  no_log: true
+  register: package_manifest_list
+  failed_when: >
+    (package_manifest_list.resources is not defined) or
+    (package_manifest_list.resources | length == 0)
+  when: packages | default([]) | length == 0
+- name: Get subscription data from OpenShift
+  ansible.builtin.set_fact:
+    openshift_sub_info: "{{ package_manifest_list.resources |
+                            map(attribute='status') |
+                            list }}"
+- name: Add each to list
+  ansible.builtin.set_fact:
+    packages: >
```

Comment:
This task fails for me due to it interpreting empty package list as a string.

The syntax is correct but I can't get it to run folded via `>`. 

I played around with it for a bit but only able to get it to run successfully with single line.

You may know an alternate way to run it multi-line.

Reply:
I will take a look

Reply:
I had tested this before I added the folding (due to lint error) but neglected to test afterwards. I'll have a fix shortly.

Reply:
Aha. Folding removes the need for the single quotes, which is what was making Ansible interpret it as a string.